.dockerignore

```diff
@@ -1,3 +1,6 @@
+.babelcache
+.cache
+.git
```

Comment:
These files were being sent to the Docker daemon as "context" during a build. Locally, this was ~15-20s worth of waiting fixed.